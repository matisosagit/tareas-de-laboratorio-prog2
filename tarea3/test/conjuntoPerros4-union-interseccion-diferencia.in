crearConjuntoPerros 10
insertarConjuntoPerros 0
insertarConjuntoPerros 1
insertarConjuntoPerros 2
insertarConjuntoPerros 3
insertarConjuntoPerros 6
insertarConjuntoPerros 7
insertarConjuntoPerros 8
insertarConjuntoPerros 9
imprimirConjuntoPerros
# unión con 1
unionConjuntoPerros 1 1
# intersección con 1
interseccionConjuntoPerros 1 1
# diferencia con 1
diferenciaConjuntoPerros 1 1
# intersección con 1 3 4 5 6
interseccionConjuntoPerros 5 1 3 4 5 6
# diferencia con 1 3 4 5 6
diferenciaConjuntoPerros 5 1 3 4 5 6
# unión con conjunto vacío
unionConjuntoPerros 0
# intersección con conjunto vacío
interseccionConjuntoPerros 0
# diferencia con conjunto vacío
diferenciaConjuntoPerros 0
liberarConjuntoPerros
Fin