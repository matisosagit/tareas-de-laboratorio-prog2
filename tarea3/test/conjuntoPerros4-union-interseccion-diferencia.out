1> Se creó conjunto perros con cantMax 10 exitosamente.
2> Se ejecutó 'insertarTConjuntoPerros con perro de id 0 exitosamente.
3> Se ejecutó 'insertarTConjuntoPerros con perro de id 1 exitosamente.
4> Se ejecutó 'insertarTConjuntoPerros con perro de id 2 exitosamente.
5> Se ejecutó 'insertarTConjuntoPerros con perro de id 3 exitosamente.
6> Se ejecutó 'insertarTConjuntoPerros con perro de id 6 exitosamente.
7> Se ejecutó 'insertarTConjuntoPerros con perro de id 7 exitosamente.
8> Se ejecutó 'insertarTConjuntoPerros con perro de id 8 exitosamente.
9> Se ejecutó 'insertarTConjuntoPerros con perro de id 9 exitosamente.
10> Conjunto de perros:
0 1 2 3 6 7 8 9
11> #  unión con 1.
12> Conjunto de perros unión:
0 1 2 3 6 7 8 9
13> #  intersección con 1.
14> Conjunto de perros intersección:
1
15> #  diferencia con 1.
16> Conjunto de perros diferencia:
0 2 3 6 7 8 9
17> #  intersección con 1 3 4 5 6.
18> Conjunto de perros intersección:
1 3 6
19> #  diferencia con 1 3 4 5 6.
20> Conjunto de perros diferencia:
0 2 7 8 9
21> #  unión con conjunto vacío.
22> Conjunto de perros unión:
0 1 2 3 6 7 8 9
23> #  intersección con conjunto vacío.
24> Conjunto de perros intersección:

25> #  diferencia con conjunto vacío.
26> Conjunto de perros diferencia:
0 1 2 3 6 7 8 9
27> Se ejecutó 'liberarTConjuntoPerros' exitosamente.
28> Fin.
